.github/workflows/dependency-review.yml

```diff
@@ -12,9 +12,9 @@ permissions:
 
 jobs:
   dependency-review:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
we've had problems in the past with ubuntu 22.04 and some networking issues between github/azure and momento. We can pin these to 24.04 if you like.

Reply:
Updated